src/document/types.ml

```diff
@@ -51,6 +51,13 @@ and Inline : sig
 end =
   Inline
 
+and Description : sig
+  type one = { attr : Class.t; key : Inline.t; definition : Block.t }
```

Comment:
Just a minor suggestion: I wouldn’t use `key`, rather `term` because it’s a convention (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

Reply:
That's what I had before. I used key on @Drup's [request](https://github.com/ocaml/odoc/pull/607#discussion_r583824544).

Reply:
The technical name used by the `dl` specification doesn't matter, since we don't use DL to represent those anymore.